### PR TITLE
Fixes and Improvements to NewTextureRepacker.

### PR DESCRIPTION
**Pull request by JohnnyonFlame**
## Description
This series fixes a long standing bug that was present on NewTextureRepacker, which caused subpar bin packing, and also adds the ability to change the bin sizes on both dimensions.

### Caveats
n/a

### Notes
n/a

### CODE REVIEW

File: UndertaleModTool/Scripts/Resource Repackers/NewTextureRepacker.csx [hunk 1]
```diff
@@ -289,12 +291,12 @@ int doItemGrouping(TPageItem item)
 //     return false;
 // }
 
-List<TextureAtlas> layoutPageItemList(List<TPageItem> items, int pageSize, int padding)
+List<TextureAtlas> layoutPageItemList(List<TPageItem> items, int pageSizeWidth, int pageSizeHeght, int padding)
```
Comment (VladiStep) [inline UndertaleModTool/Scripts/Resource Repackers/NewTextureRepacker.csx:294]:
```suggestion
List<TextureAtlas> layoutPageItemList(List<TPageItem> items, int pageSizeWidth, int pageSizeHeight, int padding)
```

File: UndertaleModTool/Scripts/Resource Repackers/NewTextureRepacker.csx [hunk 2]
```diff
@@ -289,12 +291,12 @@ int doItemGrouping(TPageItem item)
...
 {
     var atlas_list = new List<TextureAtlas>();
     while (items.Count > 0)
     {
-        var atlas = new TextureAtlas(pageSize, padding);
+        var atlas = new TextureAtlas(pageSizeWidth, pageSizeHeght, padding);
```
Comment (VladiStep) [inline UndertaleModTool/Scripts/Resource Repackers/NewTextureRepacker.csx:299]:
```suggestion
        var atlas = new TextureAtlas(pageSizeWidth, pageSizeHeight, padding);
```

**Review by VladiStep — CHANGES_REQUESTED** [review]

**Comment by VladiStep** [comment]
There was a typo ("pageSizeHe**i**ght").

**Comment by JohnnyonFlame** [comment]
Sorry forgot to push the fixes.

**Pull request merged**